src/interfaces/IResponse.ts

```diff
@@ -1,12 +0,0 @@
-/*
-    NOTE: 
-        I am choosing to use an interface here for simplicity.
-        You may choose to create a Type instead and that's okay!
-        https://www.typescriptlang.org/play/?#example/types-vs-interfaces
-*/ 
-export interface IResponse {
```

Comment:
The purpose of this type/interface was to pass info about the error from application code to handlers/UI response.

We can do this more minimalistically (?) with the new error middleware